### PR TITLE
update docker-compose and create Dockerfile

### PR DESCRIPTION
**Pull request by rezaei121**
yii3 packages requires php ^7.4

Problem:
    - This package requires php ^7.4 but your PHP version (7.2) does not satisfy that requirement.

| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Tests pass?   | ✔️
| Fixed issues  | comma-separated list of tickets # fixed by the PR, if any

### CODE REVIEW

File: docker-compose.yml.local [hunk 1]
```diff
@@ -2,14 +2,41 @@ version: '3'
 
 services:
 
-  php:
-    image: yiisoftware/yii2-php:7.2-apache
```
Comment (samdark) [inline docker-compose.yml.local]: Can't we use 7.4-apache?

Comment (rezaei121) [reply docker-compose.yml.local]: yiisoftware/yii2-php:7.4-apache not found.
error: manifest for yiisoftware/yii2-php:7.4-apache not found: manifest unknown: manifest unknown

Comment (xepozz) [reply docker-compose.yml.local]: @rezaei121 You don't need `yii2` images. See https://hub.docker.com/r/yiisoftware/yii-php/tags

Comment (rezaei121) [reply docker-compose.yml.local]: thank you @xepozz .
@samdark I updated pull request. new commit.

**Comment by samdark** [comment]
Merged.

**Pull request merged**